### PR TITLE
Move createView tag

### PR DESCRIPTION
**Pull request by thepinkcacao**
*nano: https://github.com/nanoPayinc/NANOPAY/pull/15161
*Description : Changed Card.js to the border from the view to display cards in the dashboard using only CardView.js without Card.js. 
*image:
image

### CODE REVIEW

File: src/foam/dashboard/view/Card.js [hunk 1]
```diff
@@ -118,9 +119,9 @@ foam.CLASS({
         end('div').
         start('div').
         addClass(this.myClass('content')).
-        tag(this.slot(function(data$currentView) {
-          return foam.u2.ViewSpec.createView(data$currentView, { data: this.cardData }, this, this.__subSubContext__);
-        })).
+//        tag(this.slot(function(data$currentView) {
```
Comment (sarthak-marwaha) [inline src/foam/dashboard/view/Card.js]:
```suggestion
tag('div', null, this.content$)
```

**Review by KernelDeimos — APPROVED** [review]

**Review by sarthak-marwaha — APPROVED** [review]

File: src/foam/dashboard/view/Card.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
 foam.CLASS({
   package: 'foam.dashboard.view',
   name: 'Card',
-  extends: 'foam.u2.View',
+  extends: 'foam.u2.Element',
```
Comment (kgrgreer) [inline src/foam/dashboard/view/Card.js]: What's the reason for this change?

Comment (KernelDeimos) [reply src/foam/dashboard/view/Card.js]: This element is now being used as a border rather than a view
